charts/gateway-operator/templates/validation-policy-dataplane.yaml

```diff
@@ -0,0 +1,95 @@
+{{- if and (.Capabilities.APIVersions.Has "admissionregistration.k8s.io/v1/ValidatingAdmissionPolicy") (.Capabilities.APIVersions.Has "admissionregistration.k8s.io/v1/ValidatingAdmissionPolicyBinding") -}}
```

Comment:
I believe we want a version guard here to deploy only for 1.5+, similarly as we have it in https://github.com/Kong/charts/blob/f25f1db76dbc1d348e42fb343489ab30a8117a0d/charts/gateway-operator/templates/deployment.yaml#L81